### PR TITLE
AP-392

### PR DESCRIPTION
**Pull request by vin-vlad**
Cleaned code a bit, created multiproject matching feature with confirmation events verification, deleted separate multiproject verification feature due to its implementation in multiproject feature, added few more html pages, edited configs

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
     "HOST_URL": "http://localhost:9090",
     "SHOW_BROWSER":"true",
     "WEBLOADER_URL":"https://media.evolv.ai/asset-manager/releases/next/webloader.js",
-    "PARTICIPANT_URL":"https://participants-newdev.evolvdev.com/",
-    "ENVIRONMENT_ID":"cecc5e6e80",
-    "UID":"45389200_1626264858193"
+    "PARTICIPANT_URL":"https://participants-stg.evolv.ai/",
```
Comment (robertsevernsentient) [inline config/default.json:5]: The nightly run should run on new dev

File: static/js/loader.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 var script = document.createElement('script');
 
 script.src = "https://media.evolv.ai/asset-manager/releases/next/webloader.js";
-script.setAttribute('data-evolv-environment', "cecc5e6e80");
-script.setAttribute('data-evolv-endpoint', "https://participants-newdev.evolvdev.com/");
-script.setAttribute('data-evolv-uid', "45389200_1626264858193");
+script.setAttribute('data-evolv-environment', "765b631737");
```
Comment (robertsevernsentient) [inline static/js/loader.js:4]: This should be deleted, and added to gitugnore so we don't get confsed that we need to configure ids here

File: tests/multiproject_test.js [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+        evolv.context.remoteContext.confirmations.forEach(element => {
+            confirmationsArray.push(element.cid);
+        })
+        return confirmationsArray;
+    });
+    //Assert
+    
+    I.waitForElement(locator[0]);
+    let cssAttr = await I.grabCssPropertyFrom(locator[0], css[0].attr);
+    I.assertContain(cssAttr, convert.keyword.rgb(css[0].value).join(', ').toString());
```
Comment (robertsevernsentient) [inline tests/multiproject_test.js:158]: We should check the non matching experiment changes are not applied

File: tests/multiproject_test.js [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+    let confirmations = await I.executeScript(async () => {
+        let confirmationsArray = [];
+        evolv.context.remoteContext.confirmations.forEach(element => {
+            confirmationsArray.push(element.cid);
+        })
+        return confirmationsArray;
+    });
+    //Assert
+   
+    I.waitForElement(locator[1]);
```
Comment (robertsevernsentient) [inline tests/multiproject_test.js:202]: check [0] does not apply

**Review by robertsevernsentient — APPROVED** [review]

File: config/default.json (hunk 1 above)
Comment (vin-vlad) [reply config/default.json:5]: resolved

File: tests/multiproject_test.js (hunk 3 above)
Comment (vin-vlad) [reply tests/multiproject_test.js:158]: resolved

**Pull request merged**